src/ref.rs

```diff
@@ -635,16 +640,20 @@ where
     B: ByteSlice,
     T: KnownLayout<PointerMetadata = usize> + Unaligned + Immutable + ?Sized,
 {
-    // TODO(#29), TODO(#871): Pick a name and make this public. Make sure to
-    // update references to this name in `#[deprecated]` attributes elsewhere.
-    #[doc(hidden)]
+    /// Constructs a `Ref` from the given bytes with DST length equal to `count`
+    /// without copying.
+    ///
+    /// This method attempts to return a `Ref` to the prefix of `source`
+    /// interpreted as a `T` with `count` trailing elements, and a reference to
+    /// the remaining bytes. If the length of `source` is not a valid size of
+    /// `T`, this returns `Err`.
```

Comment:
"Valid size of `T`" isn't the right condition when an explicit count is given.

Reply:
[Done](https://github.com/google/zerocopy/compare/f31d47afecf6f6652a2343d33ea5bc13662b1800..0c1f3c5d10b2842e836b549d7a8fbea1d5613700#diff-d3f14a3d45e76b3a701b00d92f105dbbde3fee7002742276173d74a5bab8d16cR649)